.g8/checkboxPage/app/controllers/$className$Controller.scala

```diff
@@ -0,0 +1,56 @@
+package controllers
+
+import controllers.actions._
+import forms.$className$FormProvider
+import javax.inject.Inject
+import models.Mode
+import navigation.Navigator
+import pages.$className$Page
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.$className$View
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class $className$Controller @Inject()(
```

Comment:
Still a newbie and but it appears this allows multiple routes to use the same controller. Wouldn't it be more maintainable to have explicit named controllers? So it is easier to maintain and not so coupled together? I might be wrong here so don't be harsh :)

Reply:
Arrr I see it is from the generator ignore me